cpp/arcticdb/version/symbol_list.cpp

```diff
@@ -120,10 +120,11 @@ bool SymbolList::can_update_symbol_list(const std::shared_ptr<Store>& store,
 
             if (stream_ids.size() == this->max_delta_ * 2 && !warned_expected_slowdown_) {
                 log::version().warn(
-                        "Warning - no compacted symbol list cache found. "
+                        "No compacted symbol list cache found. "
                         "`list_symbols` may take longer than expected. \n\n"
                         "See here for more information: https://docs.arcticdb.io/technical/on_disk_storage/#symbol-list-caching\n\n"
-                        "To resolve, run `list_symbols` through to completion to compact the symbol list cache.");
+                        "To resolve, run `list_symbols` through to completion to compact the symbol list cache.\n"
+                        "Note: This warning will only appear once.\n");
```

Comment:
Minor: Not a fan of line feeds in log lines, and I would definitely avoid having a line feed at the end.

Reply:
That's fair. I think you are right about he newline at the end. 

That said, as I've got a green build and as I'd like this to be in for the next release + this is a minor suggestion, I'm going to merge this PR.